jscomp/main/jsoo_main.ml

```diff
@@ -54,52 +56,49 @@ let implementation ~use_super_errors impl str : Js.Unsafe.obj =
   (* Question ?? *)
   (* let finalenv = ref Env.empty in *)
   let types_signature = ref [] in
-  if use_super_errors then (
-    Misc.Color.setup (Some Always);
-    Lazy.force Super_main.setup);
-
   try
-    Js_config.jsx_version := 3;
     (* default *)
     let ast = impl (Lexing.from_string str) in
-    let ast = Ppx_entry.rewrite_implementation ast in
+    let ast = Js_implementation.Ppx_entry.rewrite_implementation ast in
+    let ast =
+      Melange_ppxlib_ast.Of_ppxlib.copy_structure
```

Comment:
this changed a bit in #604

Reply:
I am not sure what is the right way to convert these asts now, using a lot of magic... https://github.com/melange-re/melange/pull/538/commits/529df6a7f99e8e04996370261e1b6a9db546348e